packages/block-editor/src/components/block-list/block-mobile-floating-toolbar.native.js

```diff
@@ -7,34 +7,15 @@ import { View, TouchableWithoutFeedback } from 'react-native';
  * Internal dependencies
  */
 import styles from './block-mobile-floating-toolbar.scss';
-/**
- * WordPress dependencies
- */
-import { createSlotFill } from '@wordpress/components';
-
-const { Fill, Slot } = createSlotFill( 'FloatingToolbar' );
 
-function FloatingToolbar( { children } ) {
+function FloatingToolbar( { children, forChild } ) {
 	return (
-		<Fill>
-			{ ( { innerFloatingToolbar } ) => {
-				return (
-					<TouchableWithoutFeedback>
-						<View
-						// Issue: `FloatingToolbar` placed above the first item in group is not touchable on Android.
-						// Temporary solution: Use `innerFloatingToolbar` to place `FloatingToolbar` over the first item in group.
-						// TODO: `{ top: innerFloatingToolbar ? 0 : -44 }` along with `innerFloatingToolbar` should be removed once issue is fixed.
-							style={ [ styles.floatingToolbarFill, { top: innerFloatingToolbar ? 0 : -44 } ] }
-						>{ children }
-						</View>
-					</TouchableWithoutFeedback>
-				);
-			} }
-
-		</Fill>
+		<TouchableWithoutFeedback>
+			<View style={ [ styles.floatingToolbarFill, forChild ? styles.withTop : {} ] }>
```

Comment:
we can rename floatingToolbarFill as floatingToolbar since there's no fill anymore